plugins/cypressRemarkPlugins/src/cypressConfigPluginExample/readme.md

```diff
@@ -0,0 +1,69 @@
+# cypress-config-plugin-example directive
+
+This "package" is a remark plugin that will take a triple colon directive
+(`:::cypress-config-plugin-example`) with one or two code blocks and convert it
+to a tabs with examples on how to use the plugin in TS and JS. If two code
+blocks are present, then the first will be included after the initial imports,
+and the second will be included in the `setupNodeEvents` method. If only one
+code block exists, it will be included in the `setupNodeEvents` method.
+
+It uses the `CypressConfigFileTabs` component to display each code block in its
+own tab.
+
+Usage:
+
+:::cypress-config-plugin-example
+
+```ts
```

Comment:
is there a difference in tagging these "ts" vs "typescript"? 

Can they be consistent if there isn't a functional difference? Just less to mentally parse.

Reply:
Theres not. Doesn't hurt but its good to be consistent.

Reply:
Same with js vs javascript